nexus-sdk-js/src/views/ElasticSearchView/index.ts

```diff
@@ -56,6 +56,21 @@ export interface ElasticSearchViewQueryResponse {
   took: number;
 }
 
+export interface Bucket {
+  key: string;
+  doc_count: number;
+}
+
+export interface ElasticSearchViewAggrigationResponse {
+  aggregations: {
+    [key: string]: {
+      doc_count_error_upper_bound: number;
+      sum_other_doc_count: number;
+      buckets: Bucket[];
+    };
+  };
+}
+
 class ElasticSeachView {
```

Comment:
This one also has a typo from an earlier commit, why not fix it at the same time? :-D